plugin/src/main/java/com/denizenscript/denizen/events/server/TabCompleteScriptEvent.java

```diff
@@ -41,10 +43,19 @@ public class TabCompleteScriptEvent extends BukkitScriptEvent implements Listene
 
     public TabCompleteScriptEvent() {
         registerCouldMatcher("tab complete");
+        registerSwitches("command");
     }
 
     public TabCompleteEvent event;
 
+    @Override
+    public boolean matches(ScriptPath path) {
+        if (!path.tryObjectSwitch("command", new ElementTag(getCommand()))) {
```

Comment:
Since it's already a `String`, can use `runGenericSwitchCheck` instead of constructing a new `ElementTag`

Reply:
Changed this